README.rst

```diff
@@ -26,6 +26,15 @@ Hypothesis is extremely practical and advances the state of the art of
 unit testing by some way. It's easy to use, stable, and powerful. If
 you're not using Hypothesis to test your project then you're missing out.
 
+------------------------
+Quick Start/Installation
+------------------------
+If you just want to get started:
+
+.. code-block:: sh
+pip install hypothesis
+.. code-block::
```

Comment:
I think the syntax is a bit off here. Here’s what [the rendered version](https://github.com/Alok/hypothesis-python/blob/patch-1/README.rst) looks like in my web browser:

<img width="494" alt="screen shot 2016-11-26 at 07 16 05" src="https://cloud.githubusercontent.com/assets/301220/20638768/448d490a-b3a8-11e6-99d8-7ef229f193fb.png">

Looking at the other examples in the README, I think you want something more like:

```rst
.. code-block:: sh

  pip install hypothesis
```